setup.py

```diff
@@ -14,7 +14,7 @@
     install_requires=[
         'escapism',
         'python-slugify',
-        'jupyterhub>=0.9',
+        'jupyterhub>=0.9.1',
```

Comment:
Isn't that conflict with jupyterhub version from `test_dependencies` of CI/CD workflow?

Reply:
Hmmm I'm not sure what you mean.